rdbms/src/main/java/org/spine3/server/storage/jdbc/aggregate/query/AggregateStorageQueryFactory.java

```diff
@@ -59,97 +59,113 @@ public AggregateStorageQueryFactory(DataSourceWrapper dataSource, Class<? extend
     }
 
     /**
-     * Use this method to path your custom loggers if you decide to overwrite {@link AggregateStorage}
+     * Sets custom loggers.
```

Comment:
I see only one param. It is better to just say "Sets the logger for logging exceptions during queries execution, etc.". One-line doc.